validator/src/main.rs

```diff
@@ -1438,6 +1448,22 @@ pub fn main() {
 
     let contact_debug_interval = value_t_or_exit!(matches, "contact_debug_interval", u64);
 
+    let mut supported_indexes: Vec<IndexType> = vec![];
+    for index in matches
+        .values_of("indexes")
+        .unwrap_or_default()
+        .map(|value| match value {
+            "program-id" => IndexType::ProgramId,
+            "token-owner" => IndexType::TokenOwner,
+            "token-mint" => IndexType::Mint,
```

Comment:
nits: I love naming consistency, so I humbly request this:

```rust
            "program-id" => IndexType::ProgramId,
            "spl-token-mint" => IndexType::SplTokenMint,
            "spl-token-owner" => IndexType::SplTokenOwner,
```

Notice the subtlety of reordering to align with byte layout of spl-token account. :)

Reply:
Done in https://github.com/solana-labs/solana/pull/14212/commits/d204a08432c3936cd1450cf16e1a67236e758461